### PR TITLE
Add fuzzing infrastructure.

### PR DESCRIPTION
**Pull request by 1425**
In response to issue #1534: https://www.veripool.org/issues/1534-Verilator-Fuzzer-script

### CODE REVIEW

File: nodist/fuzzer/actual_fail [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/python
+
+#Note: This assumes that this is Python 2.x
```
Comment (toddstrader) [inline nodist/fuzzer/actual_fail]: Ignoring the issue of adding a new language to the project (Wilson can comment on that), we should at least start with Python 3.  Time is almost up on 2.7:
https://pythonclock.org/

**Comment by 1425** [comment]
I've migrated the Python 2 to Python 3.  As far as using Python at all, I asked that here:

https://www.veripool.org/issues/1534-Verilator-Fuzzer-script#note-4

Although I didn't get a definitive answer.

**Comment by 1425** [comment]
Travis CI is now complaining that my github username is not in the contributor list.  Obviously a string of number is not my real name, and my name does appear in the contributors list.  Do you have any advice on how to proceed?

**Comment by wsnyder** [comment]
@toddstrader can you please investigate this t_dist_contrib issue? Thanks.

**Comment by toddstrader** [comment]
Can do.  I'm just on my phone now, so I'll investigate more later but I think it's just that this test is looking for the commiter's name in CONTRIBUTORS.  I think this also came up in the CMake PR.  I believe someone just changed their name for this repo to their actual name to pass the test.  Not sure if this was a local git config change or something done via GitHub.

**Comment by wsnyder** [comment]
Thanks, think the fundamentals are all here.
1. Please add to internals.adoc a section on how to run it.
2. Please add copyright headers at the top of every file e.g.

# DESCRIPTION: One line description of thingie......
#
# Copyright 2019-2019 by Wilson Snyder. This package is free software; you can
# redistribute it and/or modify it under the terms of either the GNU Lesser
# General Public License Version 3 or the Perl Artistic License Version 2.0.
######################################################################

3. Please use // comments in the .cpp file
4. Please put space after "#" before starting comment text.
5. Please put space around "=" in python & cpp.

If you prefer I'll do 2-5 for you.

Then looks good to go.

**Comment by toddstrader** [comment]
@1425 I believe if you just change your name to how it appears in `docs/CONTRIBUTORS` you'll pass the test.  You can change this in your `.gitconfig` or with `git config`.  This won't affect your GitHub name, but will change how you show up in the git log which is what matters for the test.

To test this I went the opposite way and changed my name so it would fail:
https://travis-ci.com/toddstrader/verilator-dev/jobs/246222215

**Comment by wsnyder** [comment]
Pushed with minor space/comment changes, thanks.

**Pull request closed without merging**